server/src/test/java/org/apache/calcite/avatica/remote/AvaticaServersForTest.java

```diff
@@ -163,8 +165,12 @@ public String getJdbcUrl(int port, Serialization serialization, String urlSuffix
     static JdbcMeta getInstance() {
       if (instance == null) {
         try {
-          instance = new JdbcMeta(CONNECTION_SPEC.url, CONNECTION_SPEC.username,
-              CONNECTION_SPEC.password);
+          Properties info = new Properties();
+          info.put(JdbcMeta.ConnectionCacheSettings.EXPIRY_DURATION.key(), "10");
+          info.put(JdbcMeta.ConnectionCacheSettings.EXPIRY_UNIT.key(), TimeUnit.SECONDS.name());
```

Comment:
Does this expire time config has any side effect with the test already ? I'm a little worry about it, maybe you can make these 2 params configurable and set them in your test case.

Reply:
ok.I will do that.THX